parsely/parselyandroid/ParselyTracker.java

```diff
@@ -210,26 +247,38 @@ public void startEngagement(String url) {
         this.engagementManager.start();
     }
 
-    /*! \brief Add an event Map to the queue.
+    /*! \brief Stop engaged time tracking.
      *
-     *  Place a data structure representing the event into the in-memory queue for later use
-     *
-     *  **Note**: Events placed into this queue will be discarded if the size of the persistent queue
-     *  store exceeds `storageSizeLimit`.
+     * Stops the engaged time tracker, sending any accumulated engaged time to Parse.ly.
+     * NOTE: This *must* be called during various Android lifecycle events like onPause or
+     * onStop. Otherwise, engaged time tracking will keep running and Parse.ly values
```

Comment:
Note: on iOS there's a [way](https://github.com/Parsely/AnalyticsSDK-iOS/blob/0761abfd50f8889fcce536b581ffe41a07c3c31d/ParselyTracker/ParselyTracker.swift#L185) to set up this behavior automatically. We treated that as a blocker for release, though it's arguable whether it actually is.

Reply:
I looked briefly at Android activity lifecycle and it looks like only the main activity has access to these events. I'll add it to my list to poke around more and see if we can make it automatic.